zebra-chain/src/lib.rs

```diff
@@ -22,6 +22,9 @@ extern crate serde_big_array;
 #[macro_use]
 extern crate bitflags;
 
+#[macro_use]
+extern crate static_assertions;
+
```

Comment:
Minor: if you're referencing a macro by path (like `static_assertions::macro!` directly or by having a `use static_assertions::macro;` in scope), then the `extern crate` statement is not necessary.

Reply:
Fixed in 1d4bf4a7, thanks!

You were also right about the `<`, I had even wrote like that at first but changed later, I got confused with `max_allocation()` being an inclusive bound